main_test.go

```diff
@@ -35,9 +62,62 @@ func TestScripts(t *testing.T) {
 	require.NoError(t, visitor.Walk())
 }
 
-func commonEnv(env *testscript.Env) error {
-	env.Setenv("HOME", "$WORK/home")
-	env.Setenv("VERSION", static.Version)
-	env.Setenv("COMMIT_SHA", static.Sha)
+func commonEnv(te *testscript.Env) error {
+	te.Setenv("HOME", "$WORK/home")
+	te.Setenv("VERSION", static.Version)
+	te.Setenv("COMMIT_SHA", static.Sha)
+
+	b, err := os.ReadFile(filepath.Join(te.WorkDir, testPortName))
```

Comment:
why do we need to read a file here? the file name is `testPortName`?

does the `envVarName` need to be dynamic? does each test get a clean env? can we use something fixed and obvious like ~ TX_TEST_PORT

Reply:
I needed a mechanism to request these optionally, and I was thinking that more advanced tests could require more than one port or DB, in which case this can easily expand to parse multiple lines :shrug: